importer/grpc_wrapper.go

```diff
@@ -0,0 +1,91 @@
+package importer
+
+import (
+	"github.com/goat-project/goat-proto-go"
+	"github.com/goat-project/goat/consumer/wrapper"
+	"github.com/golang/protobuf/ptypes/empty"
+)
+
+var (
+	emptyMessage = empty.Empty{}
+)
+
+// WrapVms wraps GRPC stream of VMs in a RecordStream
+func WrapVms(vms goat_grpc.AccountingService_ProcessVmsServer) RecordStream {
+	return grpcStreamWrapper{stream: vms.(grpcStream)}
+}
+
+// WrapIps wraps GRPC stream of IPs in a RecordStream
+func WrapIps(ips goat_grpc.AccountingService_ProcessIpsServer) RecordStream {
+	return grpcStreamWrapper{stream: ips.(grpcStream)}
+}
+
+// WrapStorages wraps GRPC stream of Storage records in a RecordStream
+func WrapStorages(storages goat_grpc.AccountingService_ProcessStoragesServer) RecordStream {
+	return grpcStreamWrapper{stream: storages.(grpcStream)}
+}
+
+type identifiable interface {
+	GetIdentifier() string
+}
+
+type grpcStream interface {
+	Recv() (identifiable, error)
+	SendAndClose(*empty.Empty) error
+}
+
+type grpcStreamWrapper struct {
+	stream grpcStream
+}
+
+func (gsw grpcStreamWrapper) ReceiveIdentifier() (string, error) {
+	identifiable, err := gsw.stream.Recv()
+	if err != nil {
+		return "", err
+	}
+
+	identifier := identifiable.GetIdentifier()
+	if identifier == "" {
+		return "", ErrFirstClientIdentifier
+	}
+
+	return identifier, nil
+}
+
+func (gsw grpcStreamWrapper) Receive() (wrapper.RecordWrapper, error) {
+	identifiable, err := gsw.stream.Recv()
```

Comment:
I know that technically it is `identifieble` interface but here it doesn't really matter that it is `identifiable` interface, you don't call `GetIdentifier()` here. So maybe name that variable differently so it make sense in this context.